### ISSUE TITLE
Issue: "Show/Hide Quick Paste Window" shortcut fails after configuring "Show/Hide Main App Window" shortcut 

### ISSUE THREAD

**Issue by 0n0mat0p03ia** [PasteBar/PasteBarApp#155]
Steps to reproduce:

1. In Settings, record shortcut for Quick Paste window. Click Done.
2. Confirm QP shortcut works.
3. In Settings, record shortcut for main window. Click Done. 
4. Press shortcut for Quick Paste window.

The QP window is not displayed. The shortcut for the main window does work, however. Issues is resolved after PB is restarted (both shortcuts work).

**Comment by kurdin** [comment]
@0n0mat0p03ia Yeah, in my case, it did not require app restart but in some cases, specially if your hotkeys already used by OS (you might don't know yet) you have to restart the app to activate OS hotkeys. Nothing we can do about that.

**Comment by 0n0mat0p03ia** [comment]
I'm using `Ctrl+Alt+Shift+Cmd+P` as the shortcut (`Ctrl+Alt+Shift+Cmd` is mapped to the caps lock key), I don't see this combination mapped anywhere else on my system. Not really an issue since it works after restarting PB.
